clients/ts/signalr/src/HubConnection.ts

```diff
@@ -198,13 +85,13 @@ export class HubConnection {
     }
 
     public stop(): Promise<void> {
-        this.logger.log(LogLevel.Trace, "Stopping HubConnection.");
+        this.logger.log(LogLevel.Debug, "Stopping HubConnection.");
 
         this.cleanupTimeout();
         return this.connection.stop();
     }
 
-    public stream<T>(methodName: string, ...args: any[]): Observable<T> {
+    public stream<T = any>(methodName: string, ...args: any[]): StreamResult<T> {
```

Comment:
default generic type?

Reply:
Yeah, it's a thing apparently.

Reply:
Should this be a `Promise<StreamResult<T>>`?